json-rpc/tests/integration_test.rs

```diff
@@ -411,7 +411,7 @@ fn create_test_cases() -> Vec<Test> {
                             "reason": "EINSUFFICIENT_BALANCE",
                             "reason_description": " The account does not hold a large enough balance in the specified currency"
                         },
-                        "location": "00000000000000000000000000000001::DiemAccount",
+                        "location": "0x1::DiemAccount",
```

Comment:
@xli: Just checking that this is ok from a json-rpc perspective. If clients are expecting to parse the address from this strong, I can change the printing back to the more verbose mode.

Reply:
It is fine